fs/metrics_sample.h

```diff
@@ -1,4 +1,9 @@
-//  SPDX-License-Identifier: Apache License 2.0 OR GPL-2.0
```

Comment:
Why change the spdx license identifier to a full license text? It has the same meaning. We want to use the spdx-licenses  for easier license classification.

Reply:
I formated the comments with my default IDE setting....will change it back